docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -1,9 +1,246 @@
+You need the following:
+
+- xref:install-clc.adoc[Hazelcast CLC installed] on your local machine
+- Two xref:cloud:ROOT:create-serverless-cluster.adoc[Serverless clusters]. For this exercise, we'll use one development and one production cluster, to show how you might use the CLC for basic administration and deployment tasks.
+- Connection credentials and security keys for both clusters.
+
+TIP: You can find all connection credentials, such as the cluster name and discovery token, and security keys in the {hazelcast-cloud} console. For details, see xref:cloud:ROOT:connect-to-cluster.adoc[]. 
+
+[[step-1-dev-config]]
+== Step 1. Configure the CLC for a Development Cluster
+
+Before you can connect the CLC to your development cluster, you need to add the cluster's security keys and connection details to the default configuration file.
+
+. Save the following security keys to a location on your local machine that's easy to find.
+
+** `ca.pem`
+** `cert.pem`
+** `key.pem`
++
+For example: `$HOME/.local/share/*dev*`.
+
+. Go to the directory `$HOME/.local/share/hz-cli` and open `config.yaml`. This is the default configuration file for the CLC.
+
+. Add the following configuration details for your development cluster:
++
+[source,yaml]
+----
+ssl:
+  enabled: true
+  capath: "/<PATH>/ca.pem"
+  certpath: "/<PATH>/cert.pem"
+  keypath: "/<PATH>/key.pem"
+  keypassword: "<DEV_KEYSTORE_PASSWORD>"
+hazelcast:
+  cluster:
+    name: "<DEV_CLUSTER_NAME>"
+    cloud:
+      token: "<DEV_DISCOVERY_TOKEN>"
+      enabled: true
+----
++
+NOTE: Different parameter values are required when connecting to clusters on Hazelcast Platform. See xref:configuration.adoc[] for full details.
+. Save your changes.
+
+[[step-2-dev-connect]]
+== Step 2. Connect the CLC to the Development Cluster
+You're ready to connect the CLC to your development cluster. Make sure that the cluster is running before you start.
+
+. From the command line, change into `$HOME/.local/share/hz-cli`.
+. Execute the following command:
++
+```bash
+hzc
+```
++
+The CLC connects to the development cluster and the command prompt displays the cluster details. For example: `hzc hazelcast-cloud@pr-3145>`.
+. Press kbd:[Ctrl+C] or type 'exit' to shut down the CLC while you complete the configuration for your production cluster.
+
+[[step-3-prod-configure]]
+== Step 3. Add Configuration for the Production Cluster
+Now add the security keys and a new configuration file for your production cluster.
+
+. Save the security keys for the production cluster to a separate directory on your local machine. For example: `$HOME/.local/share/*prod*`.
+. Return to `$HOME/.local/share/hz-cli`, make a copy of `config.yaml` and save it to the same directory. Give the new file a unique name. For example, `config_prod.yaml`.
+. Open the new configuration file and update the following parameters with values for your production cluster.
++
+[source,yaml]
+----
+ssl:
+  enabled: true
+  capath: "/<PATH>/ca.pem"
```

Comment:
Update the format of these placeholder as mentioned in the development cluster section (`$NAME_OF_PLACEHOLDER`)

Reply:
Updated as requested.